examples/make_assets/structure_viz.py

```diff
@@ -34,21 +34,22 @@
 struct_mp_ids = ("mp-19017", "mp-12712")
 structure_dir = f"{TEST_FILES}/structures"
 
-if not os.environ.get("GITHUB_ACTIONS"):
-    os.makedirs(structure_dir, exist_ok=True)
-    for mp_id in struct_mp_ids:
+os.makedirs(structure_dir, exist_ok=True)
+for mp_id in struct_mp_ids:
+    struct_file = f"{structure_dir}/{mp_id}.json.gz"
+    if not os.path.exists(struct_file):
```

Comment:
`os.path.isfile` is slightly more precise and semantic

Reply:
Ah, must be a typo, should have used it